app/views/meal_plans/edit.html.erb

```diff
@@ -4,6 +4,6 @@
 
 <h1>Edit the Plan for <%= @meal_plan.prepared_on.to_fs(:long) %> </h1>
 
-<%= render 'form', meal_plan: @meal_plan %>
+<%= render partial: 'form', locals: { meal_plan: @meal_plan, recipes: @recipes } %>
```

Comment:
I decided to surface the instance variable up a level so it is easier to find for future debugging.